zeppelin-integration/src/test/scala/org/apache/zeppelin/AbstractFunctionalSuite.scala

```diff
@@ -17,14 +17,16 @@
 
 package org.apache.zeppelin
 
+import org.apache.commons.io.FileUtils
```

Comment:
Not sure why we use scala for this integration test, because I see other selenium test use java.  Using scala make zeppelin-integration need to depend on scala which seems a little unnecessary. Not a big deal, but it would better to have all selenium test written in java code.

Reply:
Got this from `zeppelin-server`, but yes agreed this looks redundant, removed it.